modules/otel-collector-components.adoc

```diff
@@ -199,6 +199,138 @@ The OpenCensus receiver provides backwards compatibility with the OpenCensus pro
 Wildcards with `+*+` are accepted under the `cors_allowed_origins`.
 To match any origin, enter only `+*+`.
 
+[id="k8scluster-receiver_{context}"]
+=== Kubernetes Cluster Receiver
+
+The Kubernetes Cluster Receiver gathers cluster metrics and entity events from the Kubernetes API server. It uses the Kubernetes API to receive information about updates. Authentication for this receiver is only supported through service accounts.
+
+.OpenTelemetry Collector custom resource with the enabled Kubernetes Cluster Receiver
+[source,yaml]
+----
+  receivers:
+    k8s_cluster:
```

Comment:
Is this block a snippet of a full CR? If so, see repo guidelines on [truncating a YAML file](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting), e.g.:
```suggestion
.OpenTelemetry Collector custom resource with the enabled Kubernetes Cluster Receiver
[source,yaml]
----
# ...
  receivers:
    k8s_cluster:
```

Reply:
Thank you. Applying.

Reply:
I also created a Jira ticket to review the rest of the docs globally.